pkg/localmetrics/localmetrics.go

```diff
@@ -85,21 +89,40 @@ func NewMetricsCollector(store cache.Cache) *MetricsCollector {
 			Help:        "Report the size of account pool cr",
 			ConstLabels: prometheus.Labels{"name": operatorName},
 		}, []string{"namespace", "pool_name"}),
-		accountReadyDuration: prometheus.NewSummary(prometheus.SummaryOpts{
-			Name:        "aws_account_operator_account_ready_duration_seconds",
+
+		// Use minute as the unit
+		accountReadyDuration: prometheus.NewHistogram(prometheus.HistogramOpts{
+			Name:        "aws_account_operator_account_ready_duration_minutes",
```

Comment:
The [naming conventions](https://prometheus.io/docs/practices/naming/) specifically discourage using anything other than "base units" (in this case "seconds"). Is there a good reason to do otherwise here?

Reply:
Thanks. I think you are totally right! I will update this metric to use seconds.

Reply:
It might be good if we can introduce this linter to ensure our metrics best practice. https://github.com/prometheus/client_golang/blob/master/prometheus/testutil/promlint/promlint.go

Reply:
I wasn't able to tell at a glance how that would work, but I like the idea.

Reply:
https://github.com/coredns/coredns/blob/master/test/metric_naming_test.go That's how Coredns integrates this. We can have it as well.

Reply:
Nice. Want to throw together a card for that, in case we don't get to it right away? (Seems like a nice-to-have, but probably edged out by more immediate priorities for the forseeable.)